lib/modules/ai_bot/commands/google_command.rb

```diff
@@ -48,22 +48,28 @@ def process(query:)
         URI("https://www.googleapis.com/customsearch/v1?key=#{api_key}&cx=#{cx}&q=#{query}&num=10")
       body = Net::HTTP.get(uri)
 
-      parse_search_json(body)
+      parse_search_json(body, query)
     end
 
-    def parse_search_json(json_data)
+    def minimize_field(result, field, max_tokens: 100)
+      data = result[field].squish
+      data = ::DiscourseAi::Tokenizer::BertTokenizer.truncate(data, max_tokens).squish if max_tokens
```

Comment:
Why do `if max_tokens` here if `max_tokens` has a default value of 100?

Reply:
oops leftover from when it was nil :)